config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
   root to: 'home#index'
   devise_for :users, controllers: { registrations: 'registrations' }
-  resources :companies
+  resources :companies do
+    get '/companies/:id', to: 'jobs#index'
```

Comment:
```suggestion
```

A better way for this would be that we'd use the companies controller for the show method. 

And then the jobs#index you can redirect from the controller to the company#show. 

It would be not intuitive to overwrite resource routes to another controller.